src/api_documentation.py

```
@@ -0,0 +1,348 @@
+                    "request": {},
+                    "response": {"status": "healthy", "uptime": 3600, "models_loaded": True}
+                }
+            }
+        }
+    
+    @api.marshal_with(api_info_response)
+    def get(self):
+        """Get API information and overview."""
+        try:
+            return self.api_info
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)